app/Http/Controllers/TrainingDash.php

```diff
@@ -221,7 +221,7 @@ public function getAcademyExamTranscript($cid) {
             'http_errors' => false
         ];
         $client = new Client();
-        $res = $client->request('GET', 'https://api.vatusa.net/v2/academy/transcript/' . $cid . '?apikey=' . Config::get('vatusa.api_key'), $req_params);
+        $res = $client->request('GET', Config::get('vatusa.base').'/v2/academy/transcript/' . $cid . '?apikey=' . Config::get('vatusa.api_key'), $req_params);
```

Comment:
See comment above - move to separate issue.

Reply:
Removed